config/manager/manager.yaml

```diff
@@ -62,6 +62,26 @@ spec:
       protocol: TCP
       port: 8888
       targetPort: 8888
+    - name: metrics
+      protocol: TCP
+      port: 8080
+      targetPort: 8080
   selector:
     control-plane: controller-manager
   type: ClusterIP
+---
+kind: ServiceMonitor
+apiVersion: monitoring.coreos.com/v1
+metadata:
+  name: k4e-operator-servicemonitor
+  namespace: system
+  labels:
```

Comment:
Any reason to label the ServiceMonitor?

Reply:
That's what I saw in the documentation and examples